terraform/modules/cloudfoundry/elb_uaa.tf

```diff
@@ -198,8 +198,32 @@ resource "aws_wafv2_web_acl" "cf_uaa_waf_core" {
   }
 
   rule {
-    name     = "CG-RegexPatternSets"
+    name = "RateLimitNonCDN"
     priority = 4
+
+    action {
+      block {}
+    }
+
+    statement {
+      rate_based_statement {
+        limit = 250000
+        aggregate_key_type = "FORWARDED_IP"
```

Comment:
If this works based on the forwarded IP (e.g. `X-Forwarded-For`), won't it _only_ apply the CDN requests? Do other parts of our stack add that header?

Reply:
I don't think so. This is the screenshot of the rule as currently configured on our dev WAF:

![Screen Shot 2022-10-25 at 10 36 36 AM](https://user-images.githubusercontent.com/1001694/197803526-f81a2302-365d-4faf-827a-458de042f8ad.png)

So telling the WAF to do rate-limiting based on the forwarded IP means it won't rate limit based on the source IP, which might be the CDN IP address, but the IP address that sent the request to the CDN.

It's worth nothing that we may be okay to rate limit even CDN requests at 250,000 requests/5 minutes, but I'm just trying to be extra careful here.